.github/workflows/wheels.yml

```diff
@@ -0,0 +1,77 @@
+name: Build wheels
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Upgrade pip
+        run: python -m pip install --upgrade pip
+
+      - name: Install celer
+        run: |
+          pip install numpy==1.21
+          pip install -e .
+
+      - name: Build source dist
+        run: |
+          python setup.py sdist
+
+      - name: Install and run cibuildwheel
```

Comment:
Unfortunately, we run into the same issue with scipy

https://github.com/mathurinm/celer/actions/runs/5621826349/job/15233331748?pr=289#step:7:787